cmd/common/image/remove.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package image
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/oci"
+)
+
+func NewRemoveCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "remove IMAGE",
+		Short:        "Remove local gadget image",
+		SilenceUsage: true,
+		Args:         cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			image := args[0]
+
+			err := oci.DeleteGadgetImage(context.TODO(), image)
+			if err != nil {
+				return fmt.Errorf("removing gadget image: %w", err)
+			}
+			return nil
```

Comment:
What about printing a message when the command works fine as done in https://github.com/inspektor-gadget/inspektor-gadget/blob/17225b68c17ae58dcff76da5dc82bd38b379abbe/cmd/common/image/tag.go#L41 ?

I find it a bit weird to run it and not get any output:

```bash
$ sudo -E ig image remove docker.io/library/foo
INFO[0000] Experimental features enabled 
```